tox.ini

```diff
@@ -34,3 +34,15 @@ commands =
     make -C docs {posargs:html} SPHINXOPTS="-j 4 -n -W"
 whitelist_externals =
     make
+
+
+[testenv:test]
+basepython = python3.6
+skip_install = true
+deps =
+    -r{toxinidir}/tests/requirements.txt
+passenv = *
```

Comment:
Required to be able to use ssh-agent

Reply:
Then rather pass specific variable names, and a comment to explain why.